src_extra/networklayer_tcp.c

```diff
@@ -35,7 +35,6 @@
 
 /* workaround a glibc bug where an integer conversion is required */
 #if !defined(_WIN32)
-# include <features.h>
```

Comment:
https://gcc.gnu.org/onlinedocs/libstdc++/libstdc++-api-4.5/a00863.html#_details

hmm it seems to be related to some included features of c++ stdlib, it came into https://github.com/open62541/open62541/commit/2ae7fef1e4ae33e94a95c6f7b653655c6ff0d328 by @jpfr, can you help us?